### DISCUSSION TITLE
I run cpp file in terminal normally but not in vscode, what's wrong?

### DISCUSSION THREAD

**Discussion by grantsimongrant** [microsoft/vscode-cpptools#10584] [Q&A]
terminal: 
![image](https://user-images.githubusercontent.com/47892240/221095195-c08ab9a4-e19b-4e69-beb6-61af5d9dc8bc.png)
vscode: 
![image](https://user-images.githubusercontent.com/47892240/221095059-65680ca8-e1e2-470f-a588-3a0ca2b9a8e9.png)
my tasks.json:
![image](https://user-images.githubusercontent.com/47892240/221095093-07471efd-c8ef-4130-88cb-7e581dffdf4b.png)

**Comment by sean-mcmanus** [comment]
What file do you have active when you're building? Are you able to build via the command line when your cwd is the same as the build task?
